lib/errors.ts

```diff
@@ -157,6 +157,24 @@ export class BalenaReleaseNotFound extends BalenaError {
 }
 BalenaReleaseNotFound.prototype.code = 'BalenaReleaseNotFound';
 
+/**
+ * @summary Balena application membership role not found
+ * @class
+ * @public
+ *
+ * @param {(String|Number)} membershipRole - application membership role name or id
```

Comment:
Last NIT:
```suggestion
 * @param {(String|Number)} applicationMembershipRole - application membership role name or id
```
W/o this the node tests fail b/c the tsc sees a jsdoc `@param` which isn't part of the constructor.